test/quantization/test_quantize_fx.py

```diff
@@ -2839,6 +2838,8 @@ def _test_model_impl(
         qgraph_script = torch.jit.script(qgraph)
         # print('quantized and scripted:', qgraph_script.graph)
 
+        import copy
+        qgraph_input = copy.deepcopy(input_value)
```

Comment:
just for my own curiousity, what breaks without the copy?

Reply:
I think we can remove it, it's added when i‘m debugging